cuegui/cuegui/CueNodeGraphQt/nodes/base.py

```diff
@@ -0,0 +1,32 @@
+from builtins import str
+from NodeGraphQt import BaseNode
+from cuegui.CueNodeGraphQt.widgets.node_property import NodeProgressBar
+
+
+
+class CueBaseNode(BaseNode):
+
+    __identifier__ = 'aswf.opencue'
+
+    NODE_NAME = 'Base'
+
+    def __init__(self, rpcObject=None):
+        super(CueBaseNode, self).__init__()
+        self.add_input(name='parent', multi_input=True, display_name=False)
+        self.add_output(name='children', multi_output=True, display_name=False)
+
+        self.__rpcObject = rpcObject
+
+    def rpcObject(self):
+        return self.__rpcObject
+
+    def set_rpcObject(self, rpcObject):
```

Comment:
PEP 8: "mixedCase is allowed only in contexts where that's already the prevailing style (e.g. threading.py), to retain backwards compatibility."

IMO, I "think" it would be best to keep to the existing opencue/cuegui mixedCase as it is the prevailing style.

Reply:
re-visiting the cuegui codebase, I think I'm going to tweak some of this to be more consistent.  

For example looking at `AbstractWidgetItem` the `self.rpcObject` attribute is set in an `update` method
```python
def update(self, rpcObject=None, parent=None):
    ...
    self.rpcObject = rpcObject
```

As the relationship between `CueBaseNode`  -> `AbstractGraphWidget` is equivalent to `AbstractWidgetItem` -> `AbstractTreeWidget`, I think it makes sense to try and match this, what do you think?

Reply:
I couldn't actually rename the method to `update` as it conflicted with the base class from `NodeGraphQt`, so have changed to be mixedCase: `setRpcObject`